src/components/Header/Header.module.scss

```diff
@@ -1 +1,61 @@
+.wrapper {
+  border-bottom: 2px solid var(--color-text-primary);
+}
+.navbar {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 17px 11px;
+}
 
+.menuBtn {
+  font-size: 2rem;
+}
+
+.logo {
+  font-size: 1.75rem;
+  font-family: var(--font-title);
+  font-weight: 800;
+}
+
+.links {
+  display: none;
+}
+
+.linksContainer {
+  padding: 0;
+}
+.linksContainer li {
+  list-style: none;
+}
+.label {
+  text-decoration: none;
+}
+
+.cta {
+  color: var(--color-semantic-action);
+}
+
+.btn {
+  padding: 0;
+  background: transparent;
+  border: none;
+}
+@media (min-width: 768px) {
+  .menuBtn {
+    display: none;
+  }
+
+  .navbar {
+    padding: 16px 40px;
+  }
+
+  .links,
+  .linksContainer {
+    padding-left: 0;
+    text-transform: capitalize;
```

Comment:
Remove this line

Reply:
Hi scott, can I get more insight on this? I'm not sure which line exactly or if it's both lines that should be removed, but
if I remove padding-left, there is a 40px padding on the left bc of the ul. Is that what we want to maintain? 

And in terms of text-transform, do we want the links to be lowercase? 

![Screenshot 2023-02-07 at 8 01 14 PM](https://user-images.githubusercontent.com/79618165/217430211-42195112-9281-4957-aaf0-39e072de3ea2.png)